content/events/2020-paris/_index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Paris
+date: '2020-10-29'
+end: '2020-10-30'
+location:
+  country: France
+  city: Paris
+  url: https://en.wikipedia.org/wiki/Paris
+  venue: "????"
+map: paris-map.png
+sched_slug: kubernetescommunitydayparis
+social:
+  twitter: 'https://twitter.com/kubedaysparis'
```

Comment:
Just an FYI- The Twitter account looks good but is missing a profile image.

Reply:
We are currently working with a designer to make a logo and a profile picture for the event.